src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -75,7 +75,15 @@ PROGRAM_INFO("L2-regularized Logistic Regression and Prediction",
     "so long as an existing logistic regression model is given with the " +
     PRINT_PARAM_STRING("input_model") + " parameter.  The output predictions "
     "from the logistic regression model may be saved with the " +
-    PRINT_PARAM_STRING("output") + " parameter."
+    PRINT_PARAM_STRING("predictions") + " parameter." + 
+    "\n\n"
+    "Note : The following parameters are deprecated and " 
+    "will be removed in mlpack 4: " + PRINT_PARAM_STRING("output") + 
+    ", " + PRINT_PARAM_STRING("output_probabilities") + 
+    "\nUse " + PRINT_PARAM_STRING("predictions") + " instead of " + 
+    PRINT_PARAM_STRING("output") + "\nUse " +
```

Comment:
We should add a period to the end of each of these lines, so instead of `"\nUse "`, maybe `".\nUse "`. :)